api/users/models.py

```diff
@@ -123,6 +127,14 @@ def __str__(self):
     def subscribe_to_mailing_list(self):
         mailer_lite.subscribe(self)
 
+    @hook(AFTER_CREATE)
+    def schedule_hubspot_tracking(self) -> None:
+        if settings.ENABLE_HUBSPOT_LEAD_TRACKING:
+            track_hubspot_lead_without_organisation.delay(
+                kwargs={"user_id": self.id},
+                delay_until=timezone.now() + timedelta(minutes=30),
```

Comment:
This 30 minute number was chosen fairly arbitrarily - I wonder if I should add it to a constants file somewhere with a description of what it is so that we can easily change it in the future?

Reply:
Yeah either in settings or as a feature flag with a default value set to 30 minutes.

Reply:
Done.